File: js/comparacao-de-inteiro.js
//Escreva um algoritmo que leia dois números inteiros e compare-os, mostrando
//na tela uma das mensagens abaixo:
//- O primeiro valor é o maior
//- O segundo valor é o maior
//- Não existe valor maior, os dois são iguais

let verificarNumeroInteiro = () => {
    let valor01 = parseInt(prompt("Por favor digitem o primeiro valor: "))
    let valor02 = parseInt(prompt("Por favor repita o processo digitando um segundo valor: "))
    if(valor01 > valor02){
        console.log("O primeiro valor é o maior")
    }else if (valor01 === valor02){
        console.log("Não existe valor maior, os dois são iguais")
    }else {
        console.log("O segundo valor é o maior")
    }
}
verificarNumeroInteiro()

File: python/comparacao-de-inteiro.py
"""
Escreva um algoritmo que leia dois números inteiros e compare-os, mostrando
na tela uma das mensagens abaixo:
- O primeiro valor é o maior
- O segundo valor é o maior
- Não existe valor maior, os dois são iguais
"""


def verificar_numero_inteiro():
    valor01 = int(input("Por favor digitem o primeiro valor: "))
    valor02 = int(
        input("Por favor repita o processo digitando um segundo valor: "))

    if valor01 > valor02:
        print("O primeiro valor é o maior")
    elif valor02 == valor01:
        print("Não existe valor maior, os dois são iguais")
    else:
        print("O segundo valor é maior")


verificar_numero_inteiro()
